### ISSUE TITLE
A11y Bug:  Time picker issues

### ISSUE THREAD

**Issue by joasmi723** [cengage/react-magma#270]
The time picker is made up of multiple input elements -- a smaller input for the hour and minute, and a larger input in the background to control the styling.  The background input does not have a label, and is therefore throwing an error in a11y checkers.  

We probably need to experiment to figure out how best to solve this.  We might be able to resolve this by adding `role="presentation"` to the input, by removing that input element entirely, or by simply adding in the label text as a label.  

Furthermore, there is some inconsistent screen reader behavior in Mac Voice for the individual inputs.  Sometimes the overall name of the input gets read, but sometimes it doesn't.  Similar issue exists for helper and error messages.
